packages/core/src/domain/ai-brief/generate.ts

```diff
@@ -64,3 +70,87 @@ export async function generateAiBriefBundleEntry(
 
   return undefined;
 }
+
+async function uploadAiBriefToS3(
+  aiBriefFhirResource: Binary,
+  cxId: string,
+  patientId: string,
+  log: typeof console.log
+) {
+  try {
+    const aiBriefWrappedInBundle = buildBundleFromResources({
+      resources: [aiBriefFhirResource],
+    });
+    const s3Utils = new S3Utils(Config.getAWSRegion());
+    const key = getAiBriefFilePath(cxId, patientId);
+    await s3Utils.uploadFile({
+      bucket: Config.getAiBriefBucketName(),
+      key,
+      file: Buffer.from(JSON.stringify(aiBriefWrappedInBundle)),
+    });
+  } catch (err) {
+    const msg = `Failed to upload AI Brief to S3`;
+    log(`${msg}: ${errorToString(err)}`);
+    capture.error(msg, {
+      extra: {
+        cxId,
+        patientId,
+        error: err,
+      },
+    });
+  }
+}
+
+export async function getAiBriefFromS3(
+  cxId: string,
+  patientId: string,
+  log: typeof console.log
+): Promise<Binary | undefined> {
+  try {
+    const s3Utils = new S3Utils(Config.getAWSRegion());
+    const filePath = getAiBriefFilePath(cxId, patientId);
+    const aiBrief = await s3Utils.getFileContentsAsString(Config.getAiBriefBucketName(), filePath);
+    if (!aiBrief) {
+      return undefined;
+    }
+    log(`Got AI Brief from S3`);
+    const bundle = parseFhirBundle(aiBrief) as Bundle<Binary>;
+    if (!bundle) {
+      throw new Error("Failed to parse AI Brief bundle");
+    }
+
+    const aiBriefResource = getAiBriefResourceFromBundle(bundle);
+    if (!aiBriefResource) {
+      return undefined;
+    }
+
+    return aiBriefResource;
+  } catch (err) {
+    const msg = `Failed to get AI Brief from S3`;
+    log(`${msg}: ${errorToString(err)}`);
+    capture.error(msg, {
+      extra: {
+        cxId,
+        patientId,
+        error: err,
+      },
+    });
+    return undefined;
+  }
+}
```

Comment:
Wrapping things in a large try catch is "safe" but a good way to swallow important or unexpected errors (often loud failures are better than quiet ones). What are we worried about failing, and can we JUST try catch that instead?